DXMainClient/DXGUI/Multiplayer/CnCNet/CnCNetLobby.cs

```diff
@@ -781,6 +782,17 @@ private void SetLogOutButtonText()
         private void BtnJoinGame_LeftClick(object sender, EventArgs e) => JoinSelectedGame();
 
         private void LbGameList_DoubleLeftClick(object sender, EventArgs e) => JoinSelectedGame();
+        
+        private void LbGameList_RightClick(object sender, EventArgs e)
+        {
+            lbGameList.SelectedIndex = lbGameList.HoveredIndex;
+
+            var listedGame = (HostedCnCNetGame)lbGameList.SelectedItem?.Tag;
+            if (listedGame == null)
+                return;
+
+            globalContextMenu.Show(listedGame.HostName, GetCursorPoint());
+        }
```

Comment:
Not sure if that if that is all the needed code?

Reply:
Literally does the same as this: https://github.com/CnCNet/xna-cncnet-client/blob/977a8e29858a0ecdd850efa5c7757add95cac06b/DXMainClient/DXGUI/Multiplayer/CnCNet/CnCNetLobby.cs#L681-L693